packages/pages/src/dev/dev.ts

```diff
@@ -3,31 +3,102 @@ import { CommandModule } from "yargs";
 import open from "open";
 import { ProjectFilepaths } from "../common/src/project/structure.js";
 import { devServerPort } from "./server/middleware/constants.js";
+import runSubProcess from "../util/runSubprocess.js";
 
 interface DevArgs extends Pick<ProjectFilepaths, "scope"> {
   local?: boolean;
   "prod-url"?: boolean;
   "open-browser": boolean;
+  scope?: string;
```

Comment:
You don't need most of these. We don't want to use pages dev as a passthrough of all flags that need to be passed to the underlying calls. We will have our own defaults and if a user needs to do something else then they can call that command themselves.